### ISSUE TITLE
Examples fail with ReferenceError: _WASMEXP_ is not defined

### ISSUE THREAD

**Issue by ossy-szeged** [WebAssembly/wasm-e2e#22]
$ scripts/run.py test/fizzbuzz.c
/home/oszi/wasm/wasm-e2e/test/wasm.js:46: ReferenceError: _WASMEXP_ is not defined
m = _WASMEXP_.instantiateModule(readbuffer(arguments[0]), ffi);
    ^
ReferenceError: _WASMEXP_ is not defined
    at /home/oszi/wasm/wasm-e2e/test/wasm.js:46:5

**Comment by titzer** [comment]
In V8 we've recently renamed the _WASMEXP_ object to simply "Wasm".

**Comment by titzer** [comment]
This is already updated in sexpr-wasm-prototype, so maybe just wasm-e2e needs to be updated.

**Comment by ossy-szeged** [comment]
Ah, wasm-e2e uses fixed binaryen and sexpr-wasm-prototype revision with trunk v8. It isn't good at all.
I tried to use trunk binaryen and sexpr-wasm-prototype too, but didn't manage to get a working version.

Is it documented anywhere which revisions of these tools are compatible with each other?

**Comment by dschuff** [comment]
wasm-e2e hasn't really been updated in a while. If you want to test integration between v8, sexpr-wasm, binaryen, emscripten, and LLVM, it might be easier to try the continuous build waterfall script at https://github.com/WebAssembly/waterfall which does something similar.

**Comment by binji** [comment]
Yeah, maybe we should remove this repo? No one is updating it, and it is confusing when people find it and nothing works.

**Comment by jfbastien** [comment]
I'm not a fan of deleting things from the Internet. I'd rather have a warning :)
